src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -26,6 +26,7 @@ set MSBUILD_PROJECT_CONTENT= ^
   ^^^<ItemGroup^^^> ^
     ^^^<PackageReference Include=^"MicroBuild.Core^" Version=^"%MICROBUILD_VERSION%^" /^^^> ^
     ^^^<PackageReference Include=^"Microsoft.Net.Compilers^" Version=^"%ROSLYNCOMPILERS_VERSION%^" /^^^> ^
+    ^^^<PackageReference Include=^"Microsoft.NETCore.Compilers^" Version=^"%ROSLYNCOMPILERS_VERSION%^" /^^^> ^
```

Comment:
Do we also need to copy this over to somewhere?  (look for "Copy Roslyn Compilers Over to ToolRuntime" later in this file)

Reply:
No, we can just reference it directly from the packages folder as is done on Unix systems.

Reply:
I'll double check I didnt miss something, however.

Reply:
Yes. Looks like this is fine. Unix (`init-tools.sh`) does no such copying and everything looks to work correctly locally.

I imagine it would be possible to not copy for the desktop case (and just reference the files directly) as well (it might make some logic simpler), but that would be a separate PR.